packages/ncids-css/scss/components/nci-header/_nci-extended.scss

```diff
@@ -8,3 +8,5 @@
 @import './internal/navigation-mobile';
 @import './internal/search';
 @import './internal/print';
+
+@import '../nci-autocomplete/nci-autocomplete';
```

Comment:
This should not be in this export. If someone wants a header without autocomplete, then they should be able to just do that...